fossunited/api/pages.py

```diff
@@ -0,0 +1,43 @@
+import frappe
+
+
+@frappe.whitelist(allow_guest=True)
+def search_foss_club(query):
+    club_list = frappe.get_all(
+        "FOSS Chapter",
+        fields=[
+            "chapter_name",
+            "route",
+            "city",
+        ],
+        filters={
+            "chapter_type": "FOSS Club",
+        },
+        or_filters=[
+            ["city", "like", "%" + query + "%"],
+            ["chapter_name", "like", "%" + query + "%"],
```

Comment:
nitpicky comment - I like `f'strings'`

```suggestion
            ["city", "like", f"%{query}%"],
            ["chapter_name", "like", f"%{query}%"],
```

Reply:
I agree too with it's usage. I will fix them in my next PR of bug fixes related to FOSS Clubs and IP. This addition I think would be a bit out of scope with this PR.